ts/Core/Chart/Chart.ts

```diff
@@ -3641,7 +3641,8 @@ class Chart {
             { inverted } = this;
 
         let hasZoomed = false,
-            displayButton: boolean|undefined;
+            displayButton: boolean|undefined,
+            isAnyAxisPanning = false;
```

Comment:
```suggestion
            isAnyAxisPanning: true | undefined;
```
We can chip off a few bytes by not setting it to false to begin with.